ansible/roles/couchdb/tasks/deploy.yml

```diff
@@ -40,18 +48,9 @@
     port: "{{ db_port }}"
     timeout: 60
 
-- name: create admin user
-  uri:
-    url: "{{ db_protocol }}://{{ ansible_host }}:{{ db_port }}/_config/admins/{{ db_username }}"
-    method: PUT
-    body: >
-        "{{ db_password }}"
-    body_format: json
-    status_code: 200
```

Comment:
this is no longer needed?
is that sub summed by the `env` setting above?

Reply:
Yes, the admin user can be created with the env as well.